build.gradle

```diff
@@ -115,6 +115,7 @@ ext {
       project(':h2o-ext-jython-cfunc'),
       project(':h2o-hive'),
       project(':h2o-security'),
+      project(':h2o-logger'),      
```

Comment:
please don't forget to add to published projects as well

Reply:
added